docs/api/window-open.md

```diff
@@ -1,34 +1,57 @@
-# `window.open` Function
-
-> Open a new window and load a URL.
-
-When `window.open` is called to create a new window in a web page, a new instance
-of [`BrowserWindow`](browser-window.md) will be created for the `url` and a proxy will be returned
-to `window.open` to let the page have limited control over it.
-
-The proxy has limited standard functionality implemented to be
-compatible with traditional web pages. For full control of the new window
-you should create a `BrowserWindow` directly.
-
-The newly created `BrowserWindow` will inherit the parent window's options by
-default. To override inherited options you can set them in the `features`
-string.
+# Opening windows from the renderer
+
+Often, developers will want to control how windows are created from trusted or
+untrusted content within a renderer. Windows can be created from the renderer in two ways:
+
+- clicking on links or submitting forms adorned with `target=_blank`
+- JavaScript calling `window.open()`
+
+By default, this results in the creation of a
+[`BrowserWindowProxy`](browser-window-proxy.md), a light wrapper around
+`BrowserWindow`.
+
+However, when the `contextIsolation` or `sandbox` (or directly,
```

Comment:
This `contextIsolation` comment is incorrect, only `nativeWindowOpen` or `sandbox` (which implies `nativeWindowOpen`) will result in a real Window instance